test/fuzz/transformation_add_dead_break_test.cpp

```diff
@@ -1940,28 +1940,35 @@ TEST(TransformationAddDeadBreakTest, PhiInstructions) {
   // Some inapplicable transformations
   // Not applicable because there is already an edge 19->20, so the OpPhis at 20
   // do not need to be updated
-  ASSERT_FALSE(TransformationAddDeadBreak(19, 20, true, {13, 21})
-                   .IsApplicable(context.get(), fact_manager));
+  EXPECT_DEATH(TransformationAddDeadBreak(19, 20, true, {13, 21})
+                   .IsApplicable(context.get(), fact_manager),
+               "");
```

Comment:
Given my remarks above, can you change these back?

I didn't know about EXPECT_DEATH though - very cool!

Reply:
Fixed in 2a0b8d1.